### PR TITLE
fix: add transition to fund account tile

### PR DESCRIPTION
**Pull request by luciorubeens**
Minor update to add default transition attribute to the `FundAccountTile` component .

cc/ @kyranjamie @fbwoolf @beguene @He1DAr

### CODE REVIEW

**Review by kyranjamie — APPROVED** [review]

**Comment by fbwoolf** [comment]
@kyranjamie do we need to confirm if @landitus wants this added?

File: src/app/pages/fund/components/fund-account-tile.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export function FundAccountTile(props: FundAccountTileProps) {
       borderColor={color('border')}
       borderRadius="16px"
       boxShadow="0px 1px 2px rgba(0, 0, 0, 0.04)"
+      transition={transition}
```
Comment (fbwoolf) [inline src/app/pages/fund/components/fund-account-tile.tsx:34]: I would love for this prop to be alphabetized if possible?

**Comment by markmhendrickson** [comment]
@fbwoolf yep good idea, I've assigned him as a reviewer

**Comment by kyranjamie** [comment]
This adds a very minor animated transition to the tile's hover state. Feedback always welcome, but I wouldn't block merging for this.

**Pull request merged**